src/Rector/StaticCall/GetMockRector.php

```diff
@@ -61,6 +83,17 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
+        $callerType = $this->getType($node instanceof MethodCall ? $node->var : $node->class);
+
+        if (! $this->methodClassOwnerAnalyzer->isMethodClassOwner(
+            $callerType,
+            $scope,
+            'getMock',
+            'PHPUnit\Framework\TestCase'
+        )) {
+            return null;
+        }
+
```

Comment:
should can be handled by `ReflectionResolver->resolveClassReflectionSourceObject()`:

```php
$classReflection = $this->reflectionResolver->resolveClassReflectionSourceObject($node);
if ($classReflection instanceof  \PHPStan\Reflection\ClassReflection && $classReflection->getName() === 'PHPUnit\Framework\TestCase') {
    return null;
}
```

Reply:
Good tip, thanks :+1: